internal/manager/task_export.go

```diff
@@ -124,36 +138,44 @@ func (t *ExportTask) Start(ctx context.Context, wg *sync.WaitGroup) {
 		}()
 	}
 
+	if t.baseDir == "" {
+		logger.Errorf("baseDir must not be empty")
+		return
+	}
+
 	t.json = jsonUtils{
 		json: *paths.GetJSONPaths(t.baseDir),
 	}
 
+	_ = fsutil.EmptyDir(t.baseDir)
```

Comment:
This is a little dangerous. What if a user chooses a non empty dir as his stash metadata path? Or what if he selects the same directory with the generated path? On first full export everything will be deleted...
We could warn the user not to select the same dir (some users may have done that already though) or just create a new function that deletes only the json related dirs(if they are present) and do something like `paths.EmptyJSONDirs(t.baseDir)`